modules/olm-preparing-upgrade.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-upgrading-operators.adoc
+
+:_content-type: CONCEPT
+[id="olm-preparing-upgrade_{context}"]
+= Preparing for an Operator upgrade
+
+The subscription of an installed Operator specifies an update channel, which tracks and receives updates for the Operator. To upgrade the Operator to start tracking and receiving updates from a newer channel, you can change the update channel in the subscription.
```

Comment:
Suggestion: comma not needed after channel and maybe change "which" to "that"? (IBM: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#which)